.husky/set-node.sh

```
@@ -361,39 +355,38 @@ check_latest_pnpm_installed() {
 
 # 메인 실행
 main() {
-  echo "\n📦 노드 버전 검사 중..."
-
-  local os=$(get_os)
-  local current_version=$(get_current_node_version)
-  local lts_version=$(get_lts_version)
-  echo "\033[32m✅ Node 최신 LTS 버전: $(get_lts_major_version)"
+  printf "\n📦 노드 버전 검사 중...\n"
 
+  local os
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)